// ngx-bootstrap/modal/modal-container.component.ngfactory.js
/**
 * @fileoverview This file is generated by the Angular template compiler.
 * Do not edit.
 * @suppress {suspiciousCode,uselessCode,missingProperties,missingOverride}
 */
/* tslint:disable */
import * as import0 from '@angular/core';
import * as import1 from 'ngx-bootstrap/modal/modal-container.component';
import * as import2 from 'ngx-bootstrap/modal/modal-options.class';
import * as import3 from 'ngx-bootstrap/modal/bs-modal.service';
var styles_ModalContainerComponent = [];
export var RenderType_ModalContainerComponent = import0.ɵcrt({
    encapsulation: 2,
    styles: styles_ModalContainerComponent,
    data: {}
});
export function View_ModalContainerComponent_0(l) {
    return import0.ɵvid(0, [
        (l()(), import0.ɵted(null, ['\n    '])),
        (l()(), import0.ɵeld(0, null, null, 6, 'div', [[
                'role',
                'document'
            ]
        ], [[
                8,
                'className',
                0
            ]
        ], null, null, null, null)),
        (l()(), import0.ɵted(null, ['\n      '])),
        (l()(), import0.ɵeld(0, null, null, 3, 'div', [[
                'class',
                'modal-content'
            ]
        ], null, null, null, null, null)),
        (l()(), import0.ɵted(null, ['\n        '])),
        import0.ɵncd(null, 0),
        (l()(), import0.ɵted(null, ['\n      '])),
        (l()(), import0.ɵted(null, ['\n    '])),
        (l()(), import0.ɵted(null, ['\n  ']))
    ], null, function (ck, v) {
        var co = v.component;
        var currVal_0 = ('modal-dialog' + (co.config.class ? (' ' + co.config.class) : ''));
        ck(v, 1, 0, currVal_0);
    });
}
function View_ModalContainerComponent_Host_0(l) {
    return import0.ɵvid(0, [
        (l()(), import0.ɵeld(0, null, null, 1, 'modal-container', [
            [
                'class',
                'modal'
            ],
            [
                'role',
                'dialog'
            ],
            [
                'tabindex',
                '-1'
            ]
        ], null, [
            [
                null,
                'click'
            ],
            [
                'window',
                'keydown.esc'
            ]
        ], function (v, en, $event) {
            var ad = true;
            if (('click' === en)) {
                var pd_0 = (import0.ɵnov(v, 1).onClick($event) !== false);
                ad = (pd_0 && ad);
            }
            if (('window:keydown.esc' === en)) {
                var pd_1 = (import0.ɵnov(v, 1).onEsc() !== false);
                ad = (pd_1 && ad);
            }
            return ad;
        }, View_ModalContainerComponent_0, RenderType_ModalContainerComponent)),
        import0.ɵdid(122880, null, 0, import1.ModalContainerComponent, [
            import2.ModalOptions,
            import0.ElementRef,
            import3.BsModalService,
            import0.Renderer2
        ], null, null)
    ], function (ck, v) {
        ck(v, 1, 0);
    }, null);
}
export var ModalContainerComponentNgFactory = import0.ɵccf('modal-container', import1.ModalContainerComponent, View_ModalContainerComponent_Host_0, {}, {}, ['*']);
