// ngx-bootstrap/modal/modal-backdrop.component.ngfactory.js
/**
 * @fileoverview This file is generated by the Angular template compiler.
 * Do not edit.
 * @suppress {suspiciousCode,uselessCode,missingProperties,missingOverride}
 */
/* tslint:disable */
import * as import0 from '@angular/core';
import * as import1 from 'ngx-bootstrap/modal/modal-backdrop.component';
var styles_ModalBackdropComponent = [];
export var RenderType_ModalBackdropComponent = import0.ɵcrt({
    encapsulation: 2,
    styles: styles_ModalBackdropComponent,
    data: {}
});
export function View_ModalBackdropComponent_0(l) {
    return import0.ɵvid(0, [(l()(), import0.ɵted(null, [' ']))], null, null);
}
function View_ModalBackdropComponent_Host_0(l) {
    return import0.ɵvid(0, [
        (l()(), import0.ɵeld(0, null, null, 1, 'bs-modal-backdrop', [[
                'class',
                'modal-backdrop'
            ]
        ], null, null, null, View_ModalBackdropComponent_0, RenderType_ModalBackdropComponent)),
        import0.ɵdid(57344, null, 0, import1.ModalBackdropComponent, [
            import0.ElementRef,
            import0.Renderer2
        ], null, null)
    ], function (ck, v) {
        ck(v, 1, 0);
    }, null);
}
export var ModalBackdropComponentNgFactory = import0.ɵccf('bs-modal-backdrop', import1.ModalBackdropComponent, View_ModalBackdropComponent_Host_0, {}, {}, []);
